IntegrationTests/LayoutEventsTest.js

```diff
@@ -36,56 +35,75 @@ type State = {
   containerStyle?: ViewStyleProp,
 };
 
-const LayoutEventsTest = createReactClass({
-  displayName: 'LayoutEventsTest',
-  getInitialState(): State {
-    return {
-      didAnimation: false,
-    };
-  },
-  animateViewLayout: function() {
+class LayoutEventsTest extends React.Component<$FlowFixMeProps, State> {
```

Comment:
If we look at `react-native/IntegrationTests/IntegrationTestsApp.js`, it looks like this component is rendered with no props. So, we can remove this `$FlowFixMe`! Should have caught this earlier. 😅

```JavaScript
type Props = $ReadOnly<{||}>

class LayoutEventsTest extends React.Component<Props, State> {
```